src/commands/delete_command.hpp

```diff
@@ -0,0 +1,106 @@
+#ifndef DELETE_COMMAND_HPP
+#define DELETE_COMMAND_HPP
+
+#include "interactive_command_base.hpp"
+#include "command_manager.hpp"
+#include "../core/rq/RegionQueryService.hpp"
+#include "direct_command_base.hpp"
+
+class dicmdDeleteObj : public DirectCommandBase
+{
+public:
+	dicmdDeleteObj(IObjectPoolPtr ptr, QPoint pos)
+		: m_workingSet(std::dynamic_pointer_cast<WorkingSet>(ptr))
+	{
+		add_option("-point",new PointCommandOptionValue(pos));
+	}
+
+	dicmdDeleteObj(IObjectPoolPtr ptr)
```

Comment:
We need at least one empty constructor for log-replaying to be working.
LogReply-er will create "empty" object and then add add_arg according to options from text files

Reply:
dicmdCreateObj doesn't have default constructor. How does it work?